.travis.yml

```diff
@@ -1,13 +1,12 @@
 language: node_js
 node_js:
-- 8
```

Comment:
People with node 8, 7 and 6 will be able to use cloudevent released with version 12?

Reply:
It's possible, but not guaranteed. I don't *think* the SDK uses language features that are not available in 6, 7 and 8. But I question the wisdom of even attempting to support a Node.js version that is end of life. Both 6 and 8 are EOL, and 7 was never an LTS version (no odd numbered versions of Node.js should be considered LTS ever).

Reply:
Ok.

Reply:
LGTM